README.md

```diff
@@ -81,41 +81,61 @@ By default we download all the libraries from GitHub and build them locally, so
 
 ### Configuring on Windows
 
+#### Installing the dependencies / environment
 Install [MS Visual Studio Community / Enterprise](https://visualstudio.microsoft.com/vs/community/), [Strawberry perl](https://strawberryperl.com/), and [Chocolatey](https://chocolatey.org/install) if not installed already
 
-Get the libraries by running the following in powershell
+Get the libraries by running the following in `powershell`
 ```shell
+choco install pkgconfiglite
 choco install gstreamer
 choco install gstreamer-devel
 curl.exe -o C:\tools\pthreads-w32-2-9-1-release.zip ftp://sourceware.org/pub/pthreads-win32/pthreads-w32-2-9-1-release.zip
 mkdir C:\tools\pthreads-w32-2-9-1-release\
 Expand-Archive -Path C:\tools\pthreads-w32-2-9-1-release.zip -DestinationPath C:\tools\pthreads-w32-2-9-1-release
 ```
-
-Modify the path to the downloaded and unzipped PThreads in cmake in `build_windows_openssl.bat` if needed / unzipped at a path other than the one mentioned above
+#### Customizing the `.github\build_windows_openssl.bat` file
+If you unzipped at a path other than the one mentioned above / your pkg-config is in a different location, modify the path to the downloaded and unzipped PThreads in cmake in `.github\build_windows_openssl.bat`
 ```shell
-cmake -G "NMake Makefiles" -DBUILD_TEST=TRUE -DEXT_PTHREAD_INCLUDE_DIR="C:/tools/pthreads-w32-2-9-1-release/Pre-built.2/include/" -DEXT_PTHREAD_LIBRARIES="C:/tools/pthreads-w32-2-9-1-release/Pre-built.2/lib/x64/libpthreadGC2.a" ..
+cmake -G "NMake Makefiles" -DPKG_CONFIG_EXECUTABLE="D:\\gstreamer\\1.0\\x86_64\\bin\\pkg-config.exe" -DEXT_PTHREAD_INCLUDE_DIR="C:/tools/pthreads-w32-2-9-1-release/Pre-built.2/include/" -DEXT_PTHREAD_LIBRARIES="C:/tools/pthreads-w32-2-9-1-release/Pre-built.2/lib/x64/libpthreadGC2.a" ..
 ```
-Modify the path to MSVC as well in the `build_windows_openssl.bat` if needed / installed a different version / location
 
+If not using MSVC Enterprise 2022 or have MSVC in a different location modify the path to MSVC as well in the `.github\build_windows_openssl.bat`
+
+##### Enterprise
 ```shell
 call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x86_amd64
 ```
 
+##### Community
+```shell
+call "C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Auxiliary\Build\vcvarsall.bat" x86_amd64
+```
+
+##### Custom version
+```shell
+call "C:\Program Files\Microsoft Visual Studio\<your-version>\Community\VC\Auxiliary\Build\vcvarsall.bat" x86_amd64
+```
+
+#### Allowing long paths
 Allow long paths before we start the build
 ```shell
 git config --system core.longpaths true
 ```
 
 Note that if the paths are still too long (which can cause the build to fail unfortunately), we recommend renaming the folders to use shorter names and moving them to `C:/`
 
-Build the SDK
+#### Building the SDK
+
+Run the script customized above
```

Comment:
```suggestion
Run the `build_windows_openssl.bat` file script customized above
```

Reply:
Using consistent naming across different sections is important. If a reader is scanning through and they see this line, they will quickly be able to connect it to where we introduced this script in the previous "Customizing the .github\build_windows_openssl.bat file" section. We do not at all refer to it as a script there, so adding the longer name of it here too.

Also can consider removing "customized above" as the customization part is optional and not likely done by most first-time users - or can replace it with "... script which can be customized using the above instructions."

Reply:
The previous step mentions which file we were trying to customize and the step to build already mentions `.github\build_windows_openssl.bat` so I think it is clear